dashboard-for-desktop/articles/dashboard-designer/appearance-customization/coloring/coloring-concepts.md

```diff
@@ -3,61 +3,96 @@ title: Coloring Concepts
+
+## Color Mode: None 
+
+You can disable default color variation for dashboard item elements.  
+
+If you add a [TreeMap](../../designing-dashboard-items/treemap.md) to a dashboard, individual elements (titles) use different colors: 
+
+![win-coloring-treemap-default](../../../../images/win-coloring-treemap-default.png)
+
+_State_ values (Arguments) use different colors. To disable color variation, go to the Argument settings and select **Color by | None**.
+
+![win-coloring-treemap-none](../../../../images/win-coloring-treemap-none.png)
+
+Add a [Chart](../../designing-dashboard-items/chart.md) with the same Argument and Value as in the TreeMap: 
+
+![win-coloring-treemap-chart-none](../../../../images/win-coloring-treemap-chart-none.png)
+
+**Default** means **None** for chart arguments.  
+
+## Color Mode: Hue 
+
+You can enable colors in both previously added items.
+
+Set the TreeMap's color mode to **Default** or **Hue**: 
+
+![win-coloring-chart-none-treemap-hue](../../../../images/win-coloring-chart-none-treemap-hue.png)
+
+
+In the Chart settings, move _State_ from Arguments to Series. **Default** now means **Hue** in this new context. The ![ColoringIndicator](../../../../images/coloringindicator25453.png) indicator on the data item shows that color variation by hue is enabled.
+
+![win-coloring-chart-treemap-hue](../../../../images/win-coloring-chart-treemap-hue.png)
+
+Add _Category_ as a chart argument and switch to 100% Stacked View:  
+
+![win-coloring-full-stacked-chart](../../../../images/win-coloring-full-stacked-chart.png)
+
+## Use Global Color Scheme 
+
+The same _State_ data items use identical colors. Dashboard constructs a **Global Color Scheme** for this purpose. 
+
+Add a [Range Filter](../../designing-dashboard-items/range-filter.md) with the following settings: 
+
+![win-coloring-range-filter](../../../../images/win-coloring-range-filter.png)
+
+The same colors correspond to the same _State_ fields so you can associate and compare data from all dashboard items. This happens because all items use **Global Colors** (the default setting). You can see the switch to **Global Colors** in the Ribbon. Click **Edit Colors** to modify the colors used in the palette:  
+
+![win-coloring-custom-color](../../../../images/win-coloring-custom-color.png)
+
+> [!NOTE]
+> The dashboard uses the default palette. This palette contains 20 unique colors to paint dimension values/measures. For more information on how to change default colors or create a new color table, refer to the following help topic: [Customizing a Color Scheme](customizing-a-color-scheme.md). 
+
+This is how the Dashboard appears after you change the color for *Georgia* in the palette:
+
+![win-dashboard-with-global-color-scheme](../../../../images/win-dashboard-with-global-color-scheme.png)
+
+## Use Local Color Scheme 
+
+If you want to use an independent set of colors in the selected dashboard item, switch to the **Local Color Scheme**. 
```

Comment:
For my understanding, can you use multiple colors in a single selected dashboard item?

Reply:
Thank you.